libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -607,6 +609,36 @@ private List<String> getFullNamesFromCredentials(List<IdentityClaim> identityCla
                 .toList();
     }
 
+    public boolean checkUncorrelated(String userId)
```

Comment:
This appears to be a copy of the method in `EvaluateGpg45ScoresHandler` - can we replace that with this one?

It's also _very similar_ to the `vcDataCorrelates` method in `CheckExistingIdentityHandler` - do you think we could use it there as well?

Reply:
Just to clarify, do you mean CheckExistingIdentityHandler to use the same areVcsCorrelated method from UserIdentityService?

Reply:
That functionality in `EvaluateGpg45ScoresHandler` is removed in the second part of this ticket because it's incorporated into this `ProcessCriCallback` lambda, unless I'm wrong. So will just look into removing `vcDataCorrelates` (?)

Reply:
Yeah exactly